openhands/runtime/client/runtime.py

```diff
@@ -157,6 +163,9 @@ def __init__(
                 raise ValueError(
                     'Neither runtime container image nor base container image is set'
                 )
+            msg = 'Preparing container, this might several minutes...'
```

Comment:
```suggestion
            msg = 'Preparing sandbox, this might take a few minutes...'
```

Reply:
Yes, I already commented, that that is a typo :)

Reply:
Hah I actually missed the typo at first! But I think `a few` rather than `several` is accurate and maybe less frustrating.

Reply:
Done